client/src/main/java/org/spine3/base/Events.java

```diff
@@ -90,11 +91,20 @@ public static Timestamp getTimestamp(Event event) {
     }
 
     /**
-     * Creates {@code Event} instance with the passed event and context.
+     * Creates a new {@code Event} instance with the passed event message and context.
      */
+    @SuppressWarnings("OverloadedMethodsWithSameNumberOfParameters")
     public static Event createEvent(Message event, EventContext context) {
```

Comment:
The correct implementation is selected automatically, if we pass Any, then one with any is called; if we pass a message, which is not Any, then the method with msg param is called.